pkg/cmd/pkg/package_install.go

```diff
@@ -1,26 +1,160 @@
+			if !pkg.Selected {
+				fmt.Fprint(ctx.Out(), "This is a Community service. "+
+					"Community services are not tested for production environments. "+
+					"There may be bugs, incomplete features, incorrect documentation, or other discrepancies.\n")
+				link = "https://mesosphere.com/catalog-terms-conditions/#community-services"
+			}
+			fmt.Fprintf(ctx.Out(), "By Deploying, you agree to the Terms and Conditions %s\n", link)
+			if appOnly && pkg.PreInstallNotes != "" {
+				fmt.Fprintf(ctx.Out(), "%s\n", pkg.PreInstallNotes)
+			}
+
+			if !yes {
+				prompter := prompt.New(ctx.Input(), ctx.Out())
+				err = prompter.Confirm("Continue installing? [yes/no] ", "Yes")
+				if err != nil {
+					return err
+				}
+			}
+
+			if appOnly && description.Package.Marathon.V2AppMustacheTemplate != "" {
+				fmt.Fprintf(ctx.Out(), "Installing Marathon app for package [%s] version [%s]\n", packageName, pkg.Version)
+				err := c.PackageInstalls(appID, packageName, pkg.Version, optionsPath)
+				if err != nil {
+					return err
+				}
+			}
+
+			if cliOnly && isEmptyCli(pkg.Resource.Cli) && !isEmptyCommand(pkg.Command) {
+				return fmt.Errorf("unable to install CLI subcommand. PIP subcommands are no longer supported" +
+					" see: https://godoc.org/github.com/dcos/client-go/dcos#CosmosPackageCommand")
+			}
+
+			if cliOnly && !isEmptyCli(pkg.Resource.Cli) {
+				fmt.Fprintf(ctx.Out(), "Installing CLI subcommand for package [%s] version [%s]\n", packageName, pkg.Version)
+				pluginInfo, err := cosmos.CLIPluginInfo(description.Package, pluginutil.HTTPClient("").BaseURL())
+				if err != nil {
+					return fmt.Errorf("cannot get plugin info: %s", err)
+				}
+
+				var checksum plugin.Checksum
+				for _, contentHash := range pluginInfo.ContentHash {
+					switch contentHash.Algo {
+					case dcos.SHA256:
+						checksum.Hasher = sha256.New()
+						checksum.Value = contentHash.Value
+					}
+				}
+
+				cluster, err := ctx.Cluster()
+				if err != nil {
+					return fmt.Errorf("cannot get cluster: %s", err)
+				}
+
+				err = ctx.PluginManager(cluster).Install(pluginInfo.Url, &plugin.InstallOpts{
+					Name:     packageName,
+					Update:   true,
+					Checksum: checksum,
+					PostInstall: func(fs afero.Fs, pluginDir string) error {
+						pkgInfoFilepath := filepath.Join(pluginDir, "package.json")
+						pkgInfoFile, err := fs.OpenFile(pkgInfoFilepath, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0644)
+						if err != nil {
+							return err
+						}
+						defer pkgInfoFile.Close()
+						return json.NewEncoder(pkgInfoFile).Encode(description.Package)
+					},
+				})
+				if err != nil {
+					return fmt.Errorf("cannot install plugin: %s", err)
+				}
+				// See: https://github.com/dcos/dcos-cli/blob/72953d7eb2821f966e823294c4a451b071ee0f29/pkg/plugin/manager.go#L417
+				pluginName := packageName
+				if pluginInfo.Kind == "zip" {
+					pluginName = "dcos-" + pluginName
```

Comment:
Why adding this condition? The plugin name should always be packageName, as we pass it in line 111.

Reply:
It's not true. Plugin name depends on plugin kind see above comment

Reply:
We should wait for https://github.com/dcos/dcos-cli/pull/1497

Reply:
Done